catalogs/catalog-hadoop/src/test/java/com/datastrato/gravitino/catalog/hadoop/integration/test/HadoopCatalogIT.java

```diff
@@ -68,7 +68,7 @@ public class HadoopCatalogIT extends AbstractIT {
 
   @BeforeAll
   public static void setup() throws IOException {
-    containerSuite.startHiveContainer();
+    containerSuite.startHiveContainer(false, ImmutableMap.of());
```

Comment:
I suggest you can add a extra method in container suite like 

```java
startHiveContainer(boolean xxx) {
  startHiveContainer(xxx, ImmutableMap.of()
}

startHiveContainer(boolean xxx, Map<> xxx) {
    // core logic
}
```

In this way, we can avoid some unnecessary changes.

Reply:
Removed these code.